src/mugs.js

```diff
@@ -480,7 +480,7 @@ define([
             return this.parentMug && this.parentMug.isInRepeat();
         },
         supportsRichText: function() {
-            return this.options.richText && this.form.useRichText !== false;
+            return this.options.richText && this.form.noRichText === false;
```

Comment:
I wanted to keep the strict equality, though I don't know for sure that the strictness is necessary.